tests/reporting/reporting_utils.py

```diff
@@ -119,10 +119,10 @@ def finalizeForkingMarket(reportingFixture, market, finalizeByMigration, yesMigr
     reputationToken = reportingFixture.applySignature('ReputationToken', branch.getReputationToken())
 
     # The universe forks and there is now a branch where NO and YES are the respective outcomes of each
-    noBranch = reportingFixture.getChildBranch(branch, market, secondReportOutcomes)
+    noBranch = reportingFixture.publicGetChildBranch(branch, market, secondReportOutcomes)
```

Comment:
Same for these (and other calls throughout tests to `fixture.getChildBranch`).

Reply:
Same with these.